recipes/polyhedral-gravity-model/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "polyhedral-gravity-model" %}
+{% set org = "esa" %}
+{% set version = "1.1.0" %}
+{% set sha256 = "2255b2cd136cfd0e7f02bb1020f67af4ca2c99e23e4006e18077e10dc0a326ac" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  script_env:
+    - CMAKE_GENERATOR=Ninja
+    - PARALLELIZATION_DEVICE=OMP
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [linux]
+    - ninja
+    - python
+    - spdlog
+    - yaml-cpp
+    - xsimd
+    - pybind11
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
This isn't enough to test the package. Could you please add at least an import?

Reply:
Added